contracts/Bridge.sol

```diff
@@ -67,6 +67,20 @@ contract Bridge is DelphinusBridge {
         return cursor;
     }
 
+    function updateTokens(uint256[] memory tokens) public returns (uint32) {
```

Comment:
Need change name to InitTokens and add require condition to only allow it once for a never running contract. (Just like merkly root changing)
This is because we should not allow update existing tokens if the contract had been initialed.
(Imaging we change the original USDT to another token like eth, this will cause security issue and chaos in our system)